lib/fileAction.ts

```diff
@@ -21,44 +21,50 @@
  */
 
 import { Node } from './files/node'
+import { View } from './navigation/view'
 import logger from './utils/logger'
 
+export enum DefaultType {
+	DEFAULT = 'default',
+	HIDDEN = 'hidden',
```

Comment:
What's the idea behind switching this to an enum? What could be other values?

Reply:
If an action is hidden, it will never be displayed in the menu if another default action with a higher order gets registered.